tools/pyi/gen_pyi.py

```diff
@@ -680,6 +680,11 @@ def gen_pyi(declarations_path, out):
 
     write(out, 'torch/_C/__init__.pyi', TORCH_C_TYPE_STUBS, env)
     write(out, 'torch/_C/_VariableFunctions.pyi', TORCH_C_VARIABLE_FUNCTIONS_TYPE_STUBS, env)
+
+    if not os.path.exists('torch/_VF'):
+        os.mkdir('torch/_VF')
```

Comment:
Simply adding `_VF.pyi` as a file seems easier, no need for a separate directory. `torch/_C/` is a bit special and not quite the example to follow; `_VF.py` is a regular Python file.

Reply:
So, `torch/_VF.py`?

Reply:
`torch/_VF.pyi`